jupyterhub_singleuser_profiles/ui/package.json

```diff
@@ -1,39 +1,101 @@
 {
-  "name": "jupyterhub-singleuser-profiles-ui",
+  "name": "jupyterhub_singleuser_profiles_ui",
   "version": "0.1.0",
+  "description": "JupyterHub Spawner UI",
+  "author": "Red Hat",
```

Comment:
Should this be Red Hat upstream?

Reply:
We could also put `Open Data Hub` there, but I think Red Hat is fine